pkg/controller/kogitoinfra/manage_status.go

```diff
@@ -36,8 +34,8 @@ func updateBaseStatus(client *client.Client, instance *v1alpha1.KogitoInfra, err
 		log.Info("Kogito Infra successfully reconciled")
 	}
 	log.Infof("Updating kogitoInfra value with new properties : %s", instance.Name)
-	if resultErr := kubernetes.ResourceC(client).Update(instance); resultErr != nil {
-		log.Errorf("Error occurs while update kogitoInfra values", resultErr)
+	if resultErr := kubernetes.ResourceC(client).UpdateStatus(instance); resultErr != nil {
+		log.Errorf("reconciliationError occurs while update kogitoInfra values", resultErr)
```

Comment:
missing %v  to properly format the error:
`{"level":"error","T":"2020-10-20T14:04:38.626Z","logger":"kogitoinfra_controller","msg":"reconciliationError occurs while update kogitoInfra values%!(EXTRA *errors.StatusError=kogitoinfras.app.kiegroup.org \"kogito-kafka-infra\" not found)"}`